src/components/utils/api.js

```diff
@@ -19,7 +19,85 @@ export const getCommentsByArticleId = (id) => {
 };
 
 export const patchVote = (path, vote) => {
-  return articlesAPI
-    .patch(path, { inc_votes: vote })
-    .catch((err) => console.log(err));
+  return articlesAPI.patch(path, { inc_votes: vote });
+};
+
+export const errorTimeout = ({
```

Comment:
This js file should only really contain API calls, rather than functionality that deals with state